idl/hinted_handoff.idl.hh

```diff
@@ -39,6 +39,18 @@ struct sync_point_check_response {
     bool expired;
 };
 
+struct per_manager_sync_point_v1 {
+    std::vector<gms::inet_address> addresses;
+    std::vector<db::replay_position> flattened_rps;
+};
+
+struct sync_point_v1 {
+    utils::UUID host_id;
+    uint16_t shard_count;
+    db::hints::per_manager_sync_point_v1 regular_sp;
+    db::hints::per_manager_sync_point_v1 mv_sp;
```

Comment:
I would appreciate a comment explaining these names - right now I'm fresh enough with this code to mentally expand it to "materialized views sync point", but I might have a hard time doing so a couple of months later

Reply:
Done, added comments which, I hope, explain the meaning.